### DISCUSSION TITLE
why cannot run success in the Quickstart demo

### DISCUSSION THREAD

**Discussion by zhichao8756** [vercel/turborepo#2506] [Help] [answered]
when I run `npm run dev`, the browser show this error: `Error: React.Children.only expected to receive a single React element child.`.
Need help
![image](https://user-images.githubusercontent.com/24514251/198924329-84dc9356-68a9-42be-a628-4dcc2e3449fb.png)

**Comment by nathanhammond** [comment, marked as answer]
This isn't enough information for us to provide you any help. If you can provide more information we can revisit this.

**Reply by zhichao8756** [reply]
thanks, this is my package.json
![image](https://user-images.githubusercontent.com/24514251/199674524-996a9ba5-8925-4e81-b4a4-241bcbb69e95.png)
and my node version is `16.17.0`

**Reply by nathanhammond** [reply]
The issue is runtime, `package.json` doesn't get us anything. `React.Children.only expected to receive a single React element child` is a runtime error caused by not properly configuring some component you're using.
